src/appleseedmaya/exporters/meshexporter.cpp

```diff
@@ -436,6 +441,20 @@ void MeshExporter::meshAttributesToParams(renderer::ParamArray& params)
     bool isPhotonTarget = false;
     if (AttributeUtils::get(node(), "asIsPhotonTarget", isPhotonTarget))
         params.insert("photon_target", isPhotonTarget);
+
+    short rayBiasMethodIndex = 0; // MFnEnumAttr index is short
```

Comment:
The ray bias stuff doesn't work well enough in appleseed to justify exposing it in DCC apps. I would suggest to keep the code but `#ifdef`ing it out for now.